### ISSUE TITLE
Failing health check

### ISSUE THREAD

**Issue by eorituz** [raiden-network/raiden-service-bundle#107]
# Problem Description
When staring the service bundle the first time the health check for the MS container fails.
However, that seems to be an issue of timing. 
The docker logs of the corresponding service looked fine and after waiting a couple of minutes the health check passed and the service bundle started as expected.

# Versions used
Release 03.2020

## Docker logs of failing health check
```
root@t01:~/raiden-service-bundle# docker-compose up -d
Creating network "raiden-service-bundle_default" with the default driver
Pulling db (raidennetwork/raiden-service-bundle:2020.03.0rc1-db)...
2020.03.0rc1-db: Pulling from raidennetwork/raiden-service-bundle
6d28e14ab8c8: Pull complete
64da04b335f8: Pull complete
c2c855795327: Pull complete
1de39762bafa: Pull complete
e72de942a5bc: Pull complete
5ed4b3beed09: Pull complete
bc4fa92d5de6: Pull complete
881ba21508a4: Pull complete
0cbe8e2b7567: Pull complete
ed8782754083: Pull complete
718ba9e6241e: Pull complete
b83c71ab0d7f: Pull complete
5ceb63adfe8a: Pull complete
570544f2765b: Pull complete
8ebe256ab5c5: Pull complete
Digest: sha256:b413e9e452b223965352e71a696ad6dcaa5f3ad94695cf8f456ff37c61443801
Status: Downloaded newer image for raidennetwork/raiden-service-bundle:2020.03.0rc1-db
Pulling synapse (raidennetwork/raiden-service-bundle:2020.03.0rc1-synapse)...
2020.03.0rc1-synapse: Pulling from raidennetwork/raiden-service-bundle
50e431f79093: Already exists
dd8c6d374ea5: Already exists
c85513200d84: Already exists
55769680e827: Already exists
f5e195d50b88: Already exists
94cdd3612287: Already exists
e01b42ee5411: Already exists
044dee574de1: Already exists
f6d53d5a5c86: Already exists
64aae697bbbb: Pull complete
0c9b1869a347: Pull complete
19c161eedf52: Pull complete
859b8330ce13: Pull complete
34c7661b98b0: Pull complete
e5ab6673fdd9: Pull complete
Digest: sha256:6b85cf748fc1b53235390979b413403fe149c81445f471cfa0b647b6c80f4b73
Status: Downloaded newer image for raidennetwork/raiden-service-bundle:2020.03.0rc1-synapse
Pulling event_creator (raidennetwork/raiden-services-bundle:nightly-synapse)...
nightly-synapse: Pulling from raidennetwork/raiden-services-bundle
50e431f79093: Already exists
dd8c6d374ea5: Already exists
c85513200d84: Already exists
55769680e827: Already exists
f5e195d50b88: Already exists
94cdd3612287: Already exists
e01b42ee5411: Already exists
044dee574de1: Already exists
f6d53d5a5c86: Already exists
75707de9e855: Pull complete
0f992eb0f6bb: Pull complete
1ab36686f2be: Pull complete
81919e6045b2: Pull complete
89ecbb6d8346: Pull complete
48cca65fe60c: Pull complete
Digest: sha256:d9620436d031359655bc721fc6183629683c58784aaed892d4af5f1905a8fa8f
Status: Downloaded newer image for raidennetwork/raiden-services-bundle:nightly-synapse
Pulling well_known_server (raidennetwork/raiden-service-bundle:2020.03.0rc1-well_known_server)...
2020.03.0rc1-well_known_server: Pulling from raidennetwork/raiden-service-bundle
68ced04f60ab: Pull complete
28252775b295: Pull complete
a616aa3b0bf2: Pull complete
ade0be23e6ed: Pull complete
12ddbf5d93e9: Pull complete
8e9685a9636b: Pull complete
Digest: sha256:573343bf1a96a2767b169358f87e4fac9ec2e40341cdf559d2b44705a45a96e5
Status: Downloaded newer image for raidennetwork/raiden-service-bundle:2020.03.0rc1-well_known_server
Pulling room_ensurer (raidennetwork/raiden-service-bundle:2020.03.0rc1-room_ensurer)...
2020.03.0rc1-room_ensurer: Pulling from raidennetwork/raiden-service-bundle
68ced04f60ab: Already exists
d1d516f83cca: Pull complete
84e300e06912: Pull complete
4100d15544f4: Pull complete
2cb85865485e: Pull complete
725d8657159e: Pull complete
c849816963d4: Pull complete
Digest: sha256:972bef6d1c6826de85612a4abb7d7cb598e61ba1640b6973f54d439f003e6456
Status: Downloaded newer image for raidennetwork/raiden-service-bundle:2020.03.0rc1-room_ensurer
Pulling purger (raidennetwork/raiden-service-bundle:2020.03.0rc1-purger)...
2020.03.0rc1-purger: Pulling from raidennetwork/raiden-service-bundle
68ced04f60ab: Already exists
d1d516f83cca: Already exists
84e300e06912: Already exists
4100d15544f4: Already exists
2cb85865485e: Already exists
725d8657159e: Already exists
1756ab260c7b: Pull complete
7ae291862ad5: Pull complete
4cc46ff50805: Pull complete
649b4ed8dcaa: Pull complete
Digest: sha256:e6614936562b965c37ccf9b53880410e752d7cb6f43145d529655a816c830cb6
Status: Downloaded newer image for raidennetwork/raiden-service-bundle:2020.03.0rc1-purger
Pulling traefik (traefik:2.1)...
2.1: Pulling from library/traefik
aad63a933944: Pull complete
f365f1b91ebb: Pull complete
780c440cb354: Pull complete
d89f3c2e369b: Pull complete
Digest: sha256:ab83baa3d198ee5a2fd685bed96417055a2bdd271efe11a0b831d6c02436aabd
Status: Downloaded newer image for traefik:2.1
Creating raiden-service-bundle_traefik_1 ... done
Creating raiden-service-bundle_db_1      ... done
Creating raiden-service-bundle_ms_1      ... done


Creating raiden-service-bundle_synapse_1 ... done
Creating raiden-service-bundle_synchrotron_1       ... done
Creating raiden-service-bundle_synchrotron_2       ... done
Creating raiden-service-bundle_synchrotron_3       ... done
Creating raiden-service-bundle_synchrotron_4       ... done
Creating raiden-service-bundle_synchrotron_5       ... done
Creating raiden-service-bundle_synchrotron_6       ... done
Creating raiden-service-bundle_synchrotron_7       ... done
Creating raiden-service-bundle_synchrotron_8       ... done
Creating raiden-service-bundle_purger_1            ... done
Creating raiden-service-bundle_event_creator_1     ... done
Creating raiden-service-bundle_event_creator_2     ... done
Creating raiden-service-bundle_event_creator_3     ... done
Creating raiden-service-bundle_event_creator_4     ... done
Creating raiden-service-bundle_event_creator_5     ... done
Creating raiden-service-bundle_event_creator_6     ... done
Creating raiden-service-bundle_event_creator_7     ... done
Creating raiden-service-bundle_event_creator_8     ... done
Creating raiden-service-bundle_federation_reader_1 ... done
Creating raiden-service-bundle_room_ensurer_1      ... done
Creating raiden-service-bundle_user_dir_1          ... done
Creating raiden-service-bundle_user_dir_2          ... done
Creating raiden-service-bundle_user_dir_3          ... done
Creating raiden-service-bundle_user_dir_4          ... done
Creating raiden-service-bundle_user_dir_5          ... done
Creating raiden-service-bundle_user_dir_6          ... done
Creating raiden-service-bundle_user_dir_7          ... done
Creating raiden-service-bundle_user_dir_8          ... done
Creating raiden-service-bundle_federation_sender_1 ... done
Creating raiden-service-bundle_pfs_1               ... done
Creating raiden-service-bundle_well_known_server_1 ... done
Creating raiden-service-bundle_client_reader_1     ... done
Creating raiden-service-bundle_client_reader_2     ... done
Creating raiden-service-bundle_client_reader_3     ... done
Creating raiden-service-bundle_client_reader_4     ... done
Creating raiden-service-bundle_client_reader_5     ... done
Creating raiden-service-bundle_client_reader_6     ... done
Creating raiden-service-bundle_client_reader_7     ... done
Creating raiden-service-bundle_client_reader_8     ... done

ERROR: for msrc  Container "5afffafab2aa" is unhealthy.
ERROR: Encountered errors while bringing up the project.
```

**Comment by eorituz** [comment]
My best guess is that we either need to change the health check to be successful earlier or we need to add a waiting time until the health check will pass.

**Comment by palango** [comment]
@konradkonrad I remember vaguely that you told me you experienced this as well and that it goes away after some time.

**Comment by eorituz** [comment]
I think we can fix it by adding a `start-period: 300s` to the health check.
@fredo works on it already.

**Comment by ulope** [comment]
Does the MSRC really need 5min to finish starting?

The downside of having a long start_period is that it will also block any "downstream" dependent containers from starting for that amount of time.

**Comment by eorituz** [comment]
@ulope it took ~3 minutes the last time. 
So it's my best guess

**Comment by fredo** [comment]
resolved by #108
